clap_mangen/tests/snapshots/flatten_help.roff

```diff
@@ -18,9 +22,9 @@ my\-app
 Print help
 .SH SUBCOMMANDS
 .TP
-my\-app\-test(1)
+\fBtest\fR [\fB\-d \fR]... [\fB\-c \fR] [\fB\-h\fR|\fB\-\-help\fR] 
 Subcommand
 with a second line
 .TP
-my\-app\-help(1)
+\fBhelp\fR 
```

Comment:
Hmm, this surprised me at first but I guess this is what git does for `git help stash`